vulnerabilities/importers/__init__.py

```diff
@@ -59,6 +60,7 @@
     elixir_security.ElixirSecurityImporter,
     apache_tomcat.ApacheTomcatImporter,
     xen.XenImporter,
+    gsd.GSDImporter,
```

Comment:
We could merge the PR even if we do not actually run the importer just yet. I would like to merge the code as we may use it in the future.
```suggestion
```

Reply:
Done